VirtoCommerce.Storefront/Domain/Security/DenyAnonymousForStoreAuthorizationRequirement.cs

```diff
@@ -0,0 +1,33 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using VirtoCommerce.Storefront.Model;
+
+namespace VirtoCommerce.Storefront.Domain.Security
+{
+    public class DenyAnonymousForStoreAuthorizationRequirement : IAuthorizationRequirement
+    {
+        public const string PolicyName = "DenyAnonymousForStore";
+    }
+
+    public class DenyAnonymousForStoreAuthorizationHandler : AuthorizationHandler<DenyAnonymousForStoreAuthorizationRequirement>
```

Comment:
Bad naming. This handler doesn't deny anything, it allows access on certain conditions.
Rename it to something like AnonymousUserForStoreAuthorizationHandler.

Reply:
Done